src/parser.rs

```diff
@@ -440,7 +475,11 @@ impl<'a> Parser<'a> {
                         .collect::<String>()
                         != "//"
                 });
-                self.after_double_slash(remaining, scheme_type, scheme_end)
+                if let Some(after_prefix) = input.split_prefix("/".repeat(slashes_count as usize)) {
```

Comment:
That allocates a new `String` for nothing, please change this.

Reply:
Yeah let's pretend it never happened. :see_no_evil: